services/brig/deb/opt/brig/templates/de/team/email/sender.txt

```diff
@@ -1 +0,0 @@
-Wire
```

Comment:
@tiago-loureiro We are not using the sender.txt anymore.. right?

Reply:
[We do](https://github.com/wireapp/wire-server/blob/517cbda63ea4e1197c434286e87ed0c9f805fad6/services/brig/src/Brig/Team/Template.hs#L65), keep it there plz

Reply:
They all say `Wire`.. instead of reading the file.. can we just use `"Wire"` to keep things less complicated?

Reply:
Just to be clear.. that file is gone from everywhere :)

Reply:
In the context of supporting self-hosting eventually, it makes sense to not hard-code this. Whether using one sender.txt file or a configuration option is a different question.

Reply:
We can leave this as-is, no need to support it from the "translation" side (it will simply fallback to whatever the `sender.txt` is in the default locale)

Reply:
Nevermind my comment, it got removed from the default locale too.

Reply:
Danke!

Reply:
> Danke!

Hmmm, I actually meant that it got removed with _this_ PR and shouldn't :) 
I added them back in the default language https://github.com/wireapp/wire-server/pull/249/commits/f9fe250b809557626f903cda6155ce988ff41031